jetbrains-core/src/software/aws/toolkits/jetbrains/core/AwsSdkClient.kt

```diff
@@ -13,12 +13,16 @@ import software.amazon.awssdk.http.HttpExecuteRequest
 import software.amazon.awssdk.http.SdkHttpClient
 import software.amazon.awssdk.http.apache.ApacheHttpClient
 
-class AwsSdkClient : Disposable {
+interface AwsSdkClient : Disposable {
```

Comment:
I am not seeing a Mock for this, what was the reasoning for this change?

Reply:
There isn't a dedicated `MockAwsSdkClient` type, but it's mocked inline in `TelemetryServiceTest`:

```kotlin
DefaultTelemetryService(
        mock(), // AwsSdkClient
        // ...
)
```

Reply:
Ah okay

Reply:
The next iteration removes the need to mock `AwsSdkClient`, so I removed the interface and changed the class name back to `AwsSdkClient`.